events/api.py

```diff
@@ -799,6 +799,7 @@ class EventSerializer(LinkedEventsSerializer, GeoModelAPIView):
                                      view_name='language-detail', many=True, queryset=Language.objects.all())
     audience = JSONLDRelatedField(serializer=KeywordSerializer, view_name='keyword-detail',
                                   many=True, required=False, queryset=Keyword.objects.all())
+    is_recurring_super = serializers.BooleanField(required=False)
```

Comment:
Actually, we decided to make this field more usable. It will allow using the superevents for different purposes, i.e. changing the event model field to something like
```
-    is_recurring_super = models.BooleanField(default=False)
+    super_event_type = models.CharField(null=True, blank=True, choices=SUPER_EVENT_TYPES)
``` 
with
```
+    SUPER_EVENT_TYPES = (
+        ('recurring', _('Recurring')),
+        # Other types include e.g. a festival
+    )
```